web/cobrands/isleofwight/assets.js

```diff
@@ -0,0 +1,347 @@
+        },
+        asset_not_found: function() {
+           $('.category_meta_message').html('You can pick an <b class="asset-spot">' + this.fixmystreet.asset_item + '</b> from the map &raquo;');
+        }
+    }
+
+});
+
+var line_asset_defaults = $.extend(true, {}, defaults, {
+    road: true,
+    non_interactive: true,
+    stylemap: fixmystreet.assets.stylemap_invisible,
+    usrn: {
+        attribute: 'site_code',
+        field: 'site_code'
+    },
+    asset_item: 'road',
+    actions: {
+        found: function(layer, feature) {
+          if ( fixmystreet.assets.selectedFeature() ) {
+              return;
+          }
+          var id = feature.attributes.central_asset_id || '';
+          if (id !== '') {
+              var attrib = feature.attributes;
+              var asset_name = attrib.feature_type_name + '; ' + attrib.site_name + '; ' + attrib.feature_location;
+              $('.category_meta_message').html('You have selected ' + asset_name);
+          } else {
+              $('.category_meta_message').html('You can pick a ' + layer.fixmystreet.asset_item + ' from the map &raquo;');
+          }
+        },
+        not_found: function(layer) {
+           $('.category_meta_message').html('You can pick a ' + layer.fixmystreet.asset_item + ' from the map &raquo;');
+        }
+    }
+});
+
+
+var point_category_list = [
+    //"Dog Fouling",
+    "Manholes",
+    "Trees & Hedges",
+    //"Pavements/footpaths",
+    //"Drainage",
+    //"Car Parking",
+    "Street Lighting",
+    "Bus Stops",
+    //"Flyposting",
+    //"Potholes",
+    //"Street Cleaning",
+    "Bridges & Walls",
+    "Traffic Lights",
+    "Street Furniture",
+    //"Roads/Highways",
+    "Road Traffic Signs & Markings",
+    "Grass Verges & Weeds",
+    //"Flytipping",
+    //"Graffiti",
+    "Street Nameplates",
+    //"Abandoned Vehicles"
+];
+
+var line_category_list = [
+    "Dog Fouling",
+    "Pavements/footpaths",
+    "Potholes",
+    "Street Cleaning",
+    "Roads/Highways",
+    "Flytipping",
+    "Abandoned Vehicles"
+];
+
+var layer_map = {
+    "Dog Fouling": "Dog_Fouling",
+    "Trees & Hedges": "Trees_Hedges",
+    "Pavements/footpaths": "Pavements_footpaths",
+    "Car Parking": "Car_Parking",
+    "Street Lighting": "Street_Lighting",
+    "Bus Stops": "Bus_Stops",
+    "Street Cleaning": "Street_Cleaning",
+    "Bridges & Walls": "Bridges_Walls",
+    "Traffic Lights": "Traffic_Lights",
+    "Street Furniture": "Street_Furniture",
+    "Roads/Highways": "Roads_Highways",
+    "Road Traffic Signs & Markings": "Road_Traffic_Signs_Markings",
+    "Grass Verges & Weeds": "Grass_Verges_Weeds",
+    "Street Nameplates": "Street_Nameplates",
+    "Abandoned Vehicles": "Abandoned_Vehicles"
+};
+
+for (i = 0; i < point_category_list.length; i++) {
+    cat = point_category_list[i];
+    layer = layer_map[cat] || cat;
+
+    fixmystreet.assets.add($.extend(true, {}, point_asset_defaults, {
+        asset_group: cat,
+        asset_category: [
```

Comment:
If asset_group is set, asset_category isn't used, unless it's a `road: true` layer? So I think these all can be removed from point layers.

Reply:
I think in this case this is ok because of the non standard setup of the cobrand. The idea here is that you have groups and categories with the same name but they're never shown at the same time and this means that when the groups are showing they display the correct assets.

Reply:
But there is no piece of code that gets called using asset_category if asset_group is set on the object is my point, unless I've missed something?